financialaid/views_test.py

```diff
@@ -0,0 +1,75 @@
+"""
+Tests for financialaid view
+"""
+from django.core.urlresolvers import reverse
+from django.db.models.signals import post_save
+
+from factory.django import mute_signals
+from rest_framework.status import HTTP_201_CREATED, HTTP_400_BAD_REQUEST
+from rest_framework.test import APIClient
+
+from financialaid.backend_tests import FinancialAidBackendTests
+from financialaid.models import FinancialAid
+from profiles.factories import ProfileFactory
+
+
+class FinancialAidViewTests(FinancialAidBackendTests, APIClient):
+    """
+    Tests for financialaid views
+    """
+    def setUp(self):
+        super().setUp()
+        with mute_signals(post_save):
+            self.profile2 = ProfileFactory.create()
+        self.client.force_login(self.profile.user)
+        self.data = {
+            "original_currency": "USD",
+            "program_id": self.program.id,
+            "original_income": 80000
+        }
```

Comment:
The self.data is kept here because the data can be modified in the test and we don't want to encounter pass by reference issues.  The force_login was left here in case a future test case wanted to test logging in a different user for permissions reasons.  Let us know if you want us to move it to setUpTestData anyway; we're happy to do that!